runtime/doc/builtin.txt

```diff
@@ -4088,6 +4088,7 @@ getreg([{regname} [, 1 [, {list}]]])                                  *getreg()*
 
                 Parameters: ~
                   • {regname} (`string?`)
+                  • {one} (`any?`)
```

Comment:
This is a flag that enables some behavior. The docs say:

> getreg('=') returns the last evaluated value of the expression register.  (For use in maps.)
> getreg('=', 1) returns the expression itself, so that it can be restored with |setreg()|

So maybe `expr` works as a name.

```suggestion
                  • {expr} (`any?`)
```